final_prep/mod_zero_hero.rb

```diff
@@ -56,29 +74,86 @@
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
 
+scary_monster {
+  "name" => "Aries",
+  "smell" => "dog",
+  "weight" => 20,
+  "citiesDestroyed" => ["Denver", "New York", "Chicago"],
+  "luckyNumbers" => [1,23,43],
+
+  address {
+    "number" => 12423,
+    "street" => "park lane",
+    "state" => "Colorado",
+    "zip" => 80938
+  }
+}
+
 
 # Create a new class called SuperHero
 # - Your class should have the following DYNAMIC values
-#   - name 
+#   - name
 #   - super_power
-#   - age 
+#   - age
 # - Your class should have the following STATIC values
 #   - arch_nemesis, assigned to "The Syntax Error"
 #   - power_level = 100
-#   - energy_level = 50 
+#   - energy_level = 50
 
 # - Create the following class methods
 #   - say_name, should print the hero's name to the terminal
 #   - maximize_energy, should update the energy_level to 1000
 #   - gain_power, should take an argument of a number and INCREASE the power_level by that number
 
+class SuperHero
+  attr_accessor :name :super_power :age
+def initialize(name, super_power, age)
+  @name = name
+  @super_power = super_power
+  @age = age
+  @arch_nemesis = "The Syntax Error"
+  @power_level  = 100
+  @energy_level = 50
+end
+
+def say_name
+  puts name
```

Comment:
I believe you'd need a little piece of syntax in front of name to indicate which name you are using here. I wouldn't be surprised if it still worked when you called these methods, but more clear when you include the @